services/movie_session.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None,
+) -> QuerySet[MovieSession]:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        date = datetime.strptime(session_date, "%Y-%m-%d")
+        queryset = queryset.filter(
+            show_time__year=date.year,
+            show_time__month=date.month,
+            show_time__day=date.day
+        )
+    return queryset
+
+
+def get_movie_session_by_id(
+        movie_session_id: int
+) -> MovieSession:
+    try:
+        return MovieSession.objects.get(id=movie_session_id)
+    except MovieSession.DoesNotExist:
+        print(f"MovieSession with id {movie_session_id} does not exist")
```

Comment:
Instead of printing an error message when a movie session does not exist, consider raising an exception or returning `None`. Printing is not a recommended practice for error handling in production code. You could raise a `ValueError` or return `None` and handle it in the calling function.